src/components/Table/Table.tsx

```diff
@@ -6,11 +6,13 @@ export interface TableProps {
 }
 
 const Table: React.FC<TableProps> = ({ columns, rows }): JSX.Element => {
-    const tableColumn = columns.map((column) => <th>{column}</th>)
-    const tableRow = rows.map((row) => (
-        <tr>
+    const tableColumn = columns.map((column, index) => (
+        <th key={index}>{column}</th>
+    ))
+    const tableRow = rows.map((row, index) => (
+        <tr key={index}>
             {row.map((r) => (
-                <td>{r}</td>
+                <td key={index + Math.random()}>{r}</td>
```

Comment:
please change "r" to a better name, add another index with diffrent name to the second map and remove the Math.random() 
Thanks!

Reply:
Thanks, that helps a lot, change "r" to each row, and add a different index for each row instead of using random number